templates/public/download.html

```diff
@@ -24,7 +24,7 @@ <h3>Release Info</h3>
     or be <a href="https://wiki.archlinux.org/title/USB_flash_installation_medium">directly written to a USB flash drive</a>.
     It is intended for new installations only; an existing Arch Linux system
     can always be updated with <code>pacman -Syu</code>.</p>
-
+    <p>Images for installing Arch can be downloaded via <a href="#bittorrent-download">BitTorrent</a> or right here in your browser from one of the <a href="#download-mirrors">Arch HTTP(S) mirrors down below</a>.</p>
```

Comment:
No big no from my side but the information is just below, for the Bittorrent download

 The mirror link links to the [mirrors list directly](https://archlinux.org/download/#download-mirrors) which skips all the useful information about verifying an image so that isn't super useful.

What we probably want is to re-order the images but we don't want users to scroll over all the mirrors offering iso images to find out we have docker, cloud and netboot images.

Reply:
true, we want to make the user aware how to verify the download. adjusted the anchor on top to scroll to the HTTP Direct Downloads section, where the mirrors are linked.